{
    "functionName": "test_bad_walkers3",
    "className": "TestChain",
    "fileName": "/samreay_&_ChainConsumer/tests_&_test_chain.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "with pytest.raises(AssertionError):\n    Chain(self.d, self.p, self.n, walkers='5')\n"
}